src/pages/home/Home.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import banner from 'assets/images/banner.jpg'
+import Navbar from 'components/navbar/Navbar'
+import Footer from 'components/footer/Footer'
+import VideoCard from 'components/videocard/VideoCard'
+export default function Home() {
+  return (
+    <div className="home__container">
+      <Navbar />
+      <div className="main__container">
+        <img
+          src={banner}
+          alt="banner"
+          className="responsive__img center__image"
+        />
+        <h5 className="text-left home-banner">Trending Videos</h5>
+        <div className="videos-container">
+          <VideoCard />
+          <VideoCard />
+          <VideoCard />
+          <VideoCard />
```

Comment:
Any specific reason for rendering same component multiple times ?

Reply:
This is just for static data, will replace with one Video card once implement the API calls